src/library/themes/createTheme.js

```diff
@@ -28,119 +26,115 @@ const spacing_3x = pxToEm(24);
+    backgroundColor_warning_active: color.bronze_20,
+    backgroundColor_warning_focus: color.bronze_10,
+    backgroundColor_warning_hover: color.bronze_10,
+    backgroundColor_warningPrimary: color.bronze_60,
+    backgroundColor_warningPrimary_active: color.bronze_70,
+    backgroundColor_warningPrimary_focus: color.bronze_60,
+    backgroundColor_warningPrimary_hover: color.bronze_50,
 
     borderColor: color.gray_40,
-    borderColor_active: primaries.color_theme_60,
-    borderColor_focus: primaries.color_theme_60,
-    borderColor_hover: primaries.color_theme_60,
+    borderColor_theme: themeRamp.color_theme_60,
+    borderColor_theme_active: themeRamp.color_theme_70,
+    borderColor_theme_focus: themeRamp.color_theme_70,
+    borderColor_theme_hover: themeRamp.color_theme_50,
     borderColor_danger: color.red_60,
     borderColor_danger_active: color.red_70,
     borderColor_danger_focus: color.red_70,
-    borderColor_danger_hover: color.red_70,
-    borderColor_danger_muted: color.red_40,
-    borderColor_success: color.green_70,
-    borderColor_success_active: color.green_80,
-    borderColor_success_focus: color.green_80,
-    borderColor_success_hover: color.green_80,
-    borderColor_success_muted: color.green_40,
-    borderColor_warning: color.orange_60,
-    borderColor_warning_active: color.orange_70,
-    borderColor_warning_focus: color.orange_70,
-    borderColor_warning_hover: color.orange_70,
-    borderColor_warning_muted: color.orange_40,
+    borderColor_danger_hover: color.red_50,
+    borderColor_success: color.green_60,
+    borderColor_success_active: color.green_70,
+    borderColor_success_focus: color.green_70,
+    borderColor_success_hover: color.green_50,
+    borderColor_warning: color.bronze_60,
+    borderColor_warning_active: color.bronze_70,
+    borderColor_warning_focus: color.bronze_70,
+    borderColor_warning_hover: color.bronze_50,
+
+    boxShadow_1: '0 1px 2px 0 rgba(0,0,0,0.2), 0 2px 4px 0 rgba(0,0,0,0.2)',
+    boxShadow_2: '0 2px 4px 0 rgba(0,0,0,0.2), 0 4px 8px 0 rgba(0,0,0,0.2)',
+    boxShadow_3: '0 4px 8px 0 rgba(0,0,0,0.2), 0 8px 16px 0 rgba(0,0,0,0.2)',
+    boxShadow_4: '0 8px 16px 0 rgba(0,0,0,0.2), 0 20px 16px -8px rgba(0,0,0,0.2)',
+    boxShadow_5: '0 16px 24px 0 rgba(0,0,0,0.2), 0 32px 24px -16px rgba(0,0,0,0.2)',
+    boxShadow_focusInner: color.white,
 
     borderRadius_1: pxToEm(3),
 
     breakpoint_narrow: 512,
     breakpoint_medium: 768,
     breakpoint_wide: 1024,
 
-    color_black: color.black,
-
-    color_caption: color.gray_80,
-
-    ...grays,
-
-    color_h1: color.gray_100,
-    color_h2: color.gray_80,
-    color_h3: color.gray_80,
-    color_h4: color.gray_80,
-    color_h5: color.gray_100,
-    color_h6: color.gray_80,
+    color_disabled: color.gray_50,
+    color_danger: color.red_60,
+    color_danger_active: color.red_70,
+    color_danger_focus: color.red_60,
+    color_danger_hover: color.red_50,
     color_mouse: color.gray_80,
```

Comment:
what is  `mouse` in our schema?

Reply:
A "variation". See https://github.com/mineral-ui/mineral-ui/pull/596#discussion_r181189576